alibi/explainers/shap_wrappers.py

```diff
@@ -255,6 +309,30 @@ def __init__(self,
             `explanation.data['raw']['prediction']`
         seed
             Fixes the random number stream, which influences which subsets are sampled during shap value estimation.
+        distributed_opts
+            A dictionary with the following structure::
+                
+                {
+                'n_cpus': None,
+                'batch_size': None,
+                }
+            
+            The entries represent:
+                - `n_cpus`: an ``int`` representing the number of CPUs on which the input `X` to explain will be \
+                explained. If set to `None`, the code will run sequentially. 
+                - `batch_size`: and ``int`` representing how many instances should be explained on every CPU. If set to \
+                `None`, an input array is split in (roughly) equal parts and distributed across the available CPUs.
+                
+            The distributed explanation works only the `ray`_ library is installed. 
+            
+            .. _ray:
+               https://docs.ray.io/en/master/
+            
+        Raises
+        ------
+        ModuleNotFoundError
+            If the `ray` library is not installed.
```

Comment:
... and `n_cpus` is set in `distributed_opts`.

Reply:
Good shout ...